src/test/java/bio/terra/filesystem/EncodeFileTest.java

```diff
@@ -179,7 +179,7 @@ public void encodeFileBadFileId() throws Exception {
     public void encodeFileBadRowTest() throws Exception {
         StudySummaryModel studySummary = connectedOperations.createTestStudy("encodefiletest-study.json");
         String targetPath = loadFiles(studySummary.getId(), false, true);
-        String gsPath = "gs://" + dataRepoConfiguration.getIngestbucket() + "/" + targetPath;
+        String gsPath = "gs://" + testConfig.getIngestbucket() + "/" + targetPath;
 
```

Comment:
nit / maybe follow-on: Is it worth combining these using StringUtils.join() ?

Reply:
i think it would be a good idea if we had each segment of the path already in an array. also, here i think it would be confusing to have the first part be "gs:/". If we just wanted to use join for the pieces after the gs://, i'm not sure it's more readable:
        StringUtils.join(new String[] {dataRepoConfiguration.getIngestbucket(), targetPath}, '/');